src/Tribe/Integrations/Fusion/Widget_Shortcode.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Tribe\Events\Integrations\Fusion;
+
+/**
+ * Class Widget_Shortcode
+ *
+ * @since   TBD
+ *
+ * @package Tribe\Events\Pro\Integrations\Fusion
+ */
+class Widget_Shortcode {
+
+	/**
+	 * Fetches a list of widgets we will fix inside of the Fusion Core builder.
+	 *
+	 * @since TBD
+	 *
+	 * @return array
+	 */
+	public function get_widget_class_map() {
+		$classes = [
+			\Tribe\Events\Views\V2\Widgets\Widget_List::class,
+		];
+
+		/**
+		 * Filtering the widget classes we fix for out own widgets on Fusion builder/core.
+		 *
+		 * @since TBD
+		 *
+		 * @param array $classes List of classes we are
+		 */
+		$classes = apply_filters( 'tribe_events_integrations_fusion_widget_class_map', $classes );
```

Comment:
Either `cast` to array or make sure `classes` is always an array. 

```suggestion
		$classes = (array) apply_filters( 'tribe_events_integrations_fusion_widget_class_map', $classes );
```